packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -45,6 +46,11 @@ const handleError = (
   error.respond(res);
 };
 
+const useSites = (req: Request, res: Response, next: NextFunction) => {
+  req.useSites = true;
```

Comment:
I can't see where this is actually used...?

Reply:
Good point. It will be used in my next PR - I added the '/v1/weeklyReport/:organisationUnitCode/sites routes here but not their implementation